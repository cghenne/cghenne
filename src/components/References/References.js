// @flow
import React from 'react';
import './References.css';

const References = () => {
  return (
    <div className="box references">
      <h3>References</h3>
      <p>References available upon request.</p>
    </div>
  );
};

References.displayName = 'References';

export default References;
