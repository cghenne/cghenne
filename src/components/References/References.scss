@import '../../styles/utils.scss';

.references {
  grid-area: references;
}
