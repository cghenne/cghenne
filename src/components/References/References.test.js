import React from 'react';
import { shallow } from 'enzyme';
import References from './References';

describe('Reference', () => {
  const wrapper = shallow(<References />);

  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render title', () => {
    expect(wrapper.find('h3').text()).toEqual('References');
  });

  it('should render the reference paragraph', () => {
    expect(wrapper.find('p').text()).toEqual(
      'References available upon request.'
    );
  });

  it('should render the box with a class of box and references', () => {
    expect(wrapper.find('div').prop('className')).toEqual('box references');
  });
});
