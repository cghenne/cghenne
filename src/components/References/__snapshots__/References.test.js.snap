// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reference should render without crashing 1`] = `
<div
  className="box references"
>
  <h3>
    References
  </h3>
  <p>
    References available upon request.
  </p>
</div>
`;
